x-pack/plugins/secops/server/graphql/kpi_network/schema.gql.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const kpiNetworkSchema = gql`
+  type KpiNetworkData {
+    "The id of the source"
```

Comment:
`"The id of the source"`
I think this isn't the GraphQL comment you want? It usually applies to ID's. Pointing it out in case it's an oversight. Disregard if this is the comment you want for your GraphQL.